traffic_ops/traffic_ops_golang/monitoring/monitoring.go

```diff
@@ -151,27 +176,101 @@ func GetMonitoringJSON(tx *sql.Tx, cdnName string) (*Monitoring, error) {
 }
 
 func getMonitoringServers(tx *sql.Tx, cdn string) ([]Monitor, []Cache, []Router, error) {
```

Comment:
I can't quite tell exactly what's going on here, but in CDN-in-a-Box I have this edge server (extraneous fields omitted):
```json
{ "interfaces": [
	{
		"ipAddresses": [
			{
				"address": "172.19.0.4/16",
				"gateway": "172.19.0.1",
				"serviceAddress": true
			}
		],
		"maxBandwidth": null,
		"monitor": false,
		"mtu": 1500,
		"name": "eth0"
	}
]}
```
and in the monitoring payload it shows me these interfaces for the same server:
```json
{ "interfaces": [
	{
		"ipAddresses": [
			{
				"address": "172.19.0.5/16",
				"gateway": "172.19.0.1",
				"serviceAddress": true
			}
		],
		"maxBandwidth": null,
		"monitor": false,
		"mtu": 1500,
		"name": "eth0"
	},
	{
		"ipAddresses": [
			{
				"address": "172.19.0.10/16",
				"gateway": "172.19.0.1",
				"serviceAddress": true
			}
		],
		"maxBandwidth": null,
		"monitor": false,
		"mtu": 1500,
		"name": "eth0"
	},
	{
		"ipAddresses": [
			{
				"address": "172.19.0.4/16",
				"gateway": "172.19.0.1",
				"serviceAddress": true
			}
		],
		"maxBandwidth": null,
		"monitor": false,
		"mtu": 1500,
		"name": "eth0"
	},
	{
		"ipAddresses": [
			{
				"address": "172.19.0.11/16",
				"gateway": "172.19.0.1",
				"serviceAddress": true
			}
		],
		"maxBandwidth": null,
		"monitor": false,
		"mtu": 1500,
		"name": "eth0"
	}
]}
```

Reply:
I assume the ip addresses listed in the response actually exist for other servers and/or interfaces?

Reply:
I've torn it down since, but that is the correct IP range for other CiaB servers. So I can't prove it, but that's what I'd assume, yeah.

Reply:
Should be fixed, it was attaching every interface to each cache server. E.g. if I have 2 caches and 5 interfaces then both caches would have all 5.